tsconfig.base.json

```
@@ -39,6 +39,8 @@
 			],
 			"@woocommerce/base-hocs/*": [ "assets/js/base/hocs/*" ],
 			"@woocommerce/base-hooks": [ "assets/js/base/hooks" ],
+			"@woocommerce/base-interactivity": [ "assets/js/base/interactivity" ],
```

```suggestion
			"@woocommerce/interactivity": [ "assets/js/base/interactivity" ],
```

What do you think if we rename it to @woocommerce/interactivity?

---

That's a good suggestion. 👍